src/components/Settings.tsx

```diff
@@ -116,7 +116,7 @@ const Settings: React.FC<{
           <Form.List name="calendarList">
             {(fields, { add, remove }) => (<>
               {fields.map((field, index) => (
-                <Form.Item required label={index === 0 ? 'Calendars' : ''} {...(index === 0 ? {} : { wrapperCol: {offset: 7} })}>
+                <Form.Item required label={index === 0 ? 'Calendar' : ''} {...(index === 0 ? {} : { wrapperCol: {offset: 7} })}>
```

Comment:
This label is actually shared by multiple calendars, whether the calendars are more appropriate

![image](https://user-images.githubusercontent.com/17695989/158052853-a526775b-0cb3-4977-b17a-0e15d8581a18.png)

Reply:
Then either is ok. Still, I would *prefer* singular, it points to *one* (the default) calendar. Just a matter of taste, not of grammar, either way is ok, imho.